### PR TITLE
NEW branch added L5 DELETED 2 4 6 ,,,   Update day1

### PR DESCRIPTION
**Pull request by markandeyuluR**

### CODE REVIEW

File: day1 [hunk 1]
```diff
@@ -1,8 +1,5 @@
 line 1 suman2mark/monday
 
 
-line 4 added by markandeyuluR user on same branch
 
-added new line 6 .. please merge
-
-added new line 8 .. please merge
+deleted line 4,6,8 and added line 5 on NEW branch
```
Comment (suman2mark) [inline day1:5]: add line 7 on main branch

Comment (suman2mark) [reply day1:5]: okay right ??

**Review by suman2mark — COMMENTED** [review]
thanks mark

**Comment by markandeyuluR** [comment]
welcome suman

File: day1 [hunk 2]
```diff
@@ -3,3 +3,5 @@ line 1 suman2mark/monday
 
 
 deleted line 4,6,8 and added line 5 on NEW branch
+
+added line7
```
Comment (suman2mark) [inline day1:7]: add line 9

**Review by suman2mark — COMMENTED** [review]
add l9

**Comment by markandeyuluR** [comment]
okay

**Comment by markandeyuluR** [comment]
adding line 9

**Review by suman2mark — COMMENTED** [review]
okay fine with everything lets merge

**Comment by suman2mark** [comment]
are you okay with this code ?

**Pull request merged**